starter-code/src/functions-and-arrays.js

```diff
@@ -60,6 +110,14 @@ var wordsFind = [
   'disobedience'
 ];
 
+function doesWordExist(wordsFind, searchWord) {
+  for (let i = 0; i < wordsFind.length; i += 1) {
+      if (wordsFind[i].toLowerCase() === searchWord.toLowerCase()) {
+        return true;
+  } else {}
```

Comment:
Se não há condicional no `else` ou se for algo como `continue;`, não precisa colocar o `else`.